packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -93,7 +93,13 @@ class PoolState {
       address: this.address,
       l1Token: this.l1Token,
       exchangeRatePrevious,
-      ...(await this.batchRead([
+      ...(await this.batchRead<{
+        exchangeRateCurrent: BigNumber;
+        liquidityUtilizationCurrent: BigNumber;
+        liquidReserves: BigNumber;
+        pendingReserves: BigNumber;
+        utilizedReserves: BigNumber;
+      }>([
```

Comment:
The batch read function was made generic so we can properly type the results. needed to update the call in the bridge pool